### PR TITLE
update runtime deps

### PR DESCRIPTION
**Pull request by betolink**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by betolink** [comment]
@conda-forge-admin, please rerender

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/itslive-feedsto
 
 Home: https://github.com/nasa-jpl/itslive-vortex
 
-Package license: GPL-3.0
+Package license: BSD-3-Clause
```
Comment (weiji14) [inline README.md:8]: Package license is still GPL-3.0 no? https://github.com/nasa-jpl/itslive-py/blob/main/LICENSE

Comment (weiji14) [reply README.md:8]: Oops, @betolink, could you open another PR to patch the license, and maybe update the URLs from https://github.com/nasa-jpl/itslive-vortex to https://github.com/nasa-jpl/itslive-py

Comment (betolink) [reply README.md:8]: Yeah I updated that just before merging, got confused with the license from the feedstock, yes the license is still GPLv3. Thanks for noticing Wei Ji! also would you want to be on the maintainers list? (both the library and the conda-forge repos)

Comment (weiji14) [reply README.md:8]: Maybe just the conda-forge repo, I'm less familiar with what the Python client library is doing :slightly_smiling_face:

**Pull request merged**